spec.bs

```diff
@@ -18,3 +18,91 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="join-ad-interest-group">joinAdInterestGroup</h2>
+
+<xmp class="idl">
+[
+  SecureContext
+] partial interface Navigator {
+  Promise<undefined> joinAdInterestGroup(AuctionAdInterestGroup group, double durationSeconds);
+};
+</xmp>
+
+
+<h2 id="run-ad-auction">runAdAuction</h2>
+
+<xmp class="idl">
+[
+  SecureContext
+] partial interface Navigator {
+  Promise<USVString?> runAdAuction(AuctionAdConfig config);
+};
+</xmp>
+
+
+<xmp class="idl">
+dictionary AuctionAdConfig {
+  required USVString seller;
+  required USVString decisionLogicUrl;
+  USVString trustedScoringSignalsUrl;
+  sequence<USVString> interestGroupBuyers;
+  any auctionSignals;
+  any sellerSignals;
+  USVString directFromSellerSignals;
+  unsigned long long sellerTimeout;
+  unsigned short sellerExperimentGroupId;
+  record<USVString, any> perBuyerSignals;
+  record<USVString, unsigned long long> perBuyerTimeouts;
+  record<USVString, unsigned short> perBuyerGroupLimits;
+  record<USVString, unsigned short> perBuyerExperimentGroupIds;
+  record<USVString, record<USVString, double>> perBuyerPrioritySignals;
+  sequence<AuctionAdConfig> componentAuctions;
+  AbortSignal? signal;
+};

```

Comment:
Put this in the same `<xmp>` block as `runAdAuction()`.

Reply:
done